examples/with-plasmo/package.json

```diff
@@ -10,7 +10,7 @@
   "dependencies": {
     "@trpc/client": "^10.3.0",
```

Comment:
no need to update @trpc deps? (even if it's the example)

Reply:
wont update as this should only be a minor release and I dont wanna break anything